interpreter/text/parser.mly

```diff
@@ -335,7 +334,7 @@ plain_instr :
   | MEMORY_SIZE { fun c -> memory_size }
   | MEMORY_GROW { fun c -> memory_grow }
   | CONST literal { fun c -> fst (literal $1 $2) }
-  | V128_CONST SIMD_SHAPE literal_list { fun c -> fst (simd_literal $2 $3) }
+  | V128_CONST SIMD_SHAPE literal_list { fun c -> fst (simd_literal $2 $3 (at ())) }
```

Comment:
Since this is deferred by a fun, you need to let-bind the at value outside the fun, otherwise the parser state will have change already when the fun gets called.

Reply:
Good catch, thanks!